docs/learn/learn-opengov.md

```diff
@@ -222,11 +222,8 @@ emergency referendum occurring at the same time as a regular referendum (either
 council-proposed) is the only time that multiple referenda will be able to be voted on
 simultaneously.
 
-OpenGov shares the same
-{{ polkadot: <RPC network="polkadot" path="consts.democracy.votingPeriod" defaultValue={403200} filter="blocksToDays" /> :polkadot }}
-{{ kusama: <RPC network="kusama" path="consts.democracy.votingPeriod" defaultValue={100800} filter="blocksToDays" /> :kusama }}
-day eligibility period when the proposal can get approved. If not approved by then end of this
-period, the proposal is automatically rejected.
+OpenGov shares has the deciding period when the proposal is eligible to get approved. If not 
```

Comment:
I don't understand this first sentence, "OpenGov shares has the deciding period..."

Reply:
Hi Bill! The PR is now ready for review. It originally stated that the decision period was the same as the legacy democracy voting period. I updated that sentence and added links to the table with decision periods for the respective tracks.